src/main/java/com/jfrog/ide/idea/ui/webview/event/tasks/JumpToCodeTask.java

```diff
@@ -0,0 +1,22 @@
+package com.jfrog.ide.idea.ui.webview.event.tasks;
+
+import com.intellij.openapi.project.Project;
+import com.jfrog.ide.idea.inspections.JumpToCode;
+import com.jfrog.ide.idea.ui.webview.model.Location;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Represents a task that performs the "Jump to Code" action in the IDE.
+ * This task is responsible for executing the jump to code operation based on the provided project and location.
+ */
+public class JumpToCodeTask {
```

Comment:
What's the purpose of this class? Why don't you just use JumpToCode directly instead?

Reply:
The purpose behind this is to serve as a _guideline_ for upcoming tasks. In my perspective, having a `handler(IdeEvent event)` would function as the central starting point for various tasks. However, the intention is not for every task to embed its logic within the handler function. Instead, the idea is to have each task encapsulated within its own class, containing the appropriate logic.